azure-pipelines.yml

```diff
@@ -22,6 +22,8 @@ variables:
     value: 'analyzers\tests\SonarAnalyzer.UnitTest\'
   - name: UnitTestResultsPath
     value: '$(UnitTestProjectPath)\TestResults'
+  - name: UnitTestExclusionsPattern
+    value: 'analyzers/tests/SonarAnalyzer.UnitTest/TestCases/**/*'
```

Comment:
note that the path separator is different from `    value: 'analyzers\tests\SonarAnalyzer.UnitTest\'` in line 22

Reply:
I added it like this on purpose to match the documentation from: https://docs.sonarsource.com/sonarqube/latest/project-administration/analysis-scope/